src/main/java/com/inventor/email/InputConsole.java

```diff
@@ -0,0 +1,92 @@
+package com.inventor.email;
+
+import org.springframework.stereotype.Component;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class InputConsole {
+    private boolean isInput() {
+        System.out.print("Чи бажаєте ви ввести пошту? (y/n): ");
+        char answer = new Scanner(System.in).nextLine().charAt(0);
+        if(answer == 'y' || answer == 'Y'){
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
+    private static final String regex = "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$";
```

Comment:
Please move variable declarations to the top. Also, for constants you can use all caps name, like REGEX